opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -27,6 +27,16 @@ const rootSettingsFragment = graphql`
       message
     }
     platform_banner_level
+    platform_critical_alerts {
+      message
+      type
+      details {
+        groups {
+          id 
+          name 
+        }
+      }
+    }
```

Comment:
alert details shall depends on alert type... current implem is a bit rough on the edges but we had to move quickly about this.